### ISSUE TITLE
Profile picture

### ISSUE THREAD

**Issue by AlenAlbreht** [yesiman/INRAE.VINEAS#69]
I cannot add a profile picture for my profile on VINEAS. When I select the photo and click OK, the page refreshes but nothing happens. Are there any picture restrictions members should know about?

**Comment by MaddyTintinger** [comment]
Did you click on validate after loading your picture ? Maybe that is the issue ?

**Comment by AlenAlbreht** [comment]
Yes, I clicked on validate, but nothing happens. After I choose my profile picture from the pop-up window (after I click on my profile) I choose the picture, the window closes, but nothing happens. The picture doeas not apear in the left top-side corner, regardless of validating.

**Comment by yesiman** [comment]
It should work, can you send me the picture you have used.
Thanks.

Le mar. 5 oct. 2021 à 14:57, AlenAlbreht ***@***.***> a
écrit :

**Comment by AlenAlbreht** [comment]
Dear Fabrice,

I was playing with the photo attached and cannot seem to get it to work.

Best
Alen



dr. Alen Albreht
Vodja Laboratorija za prehrambeno kemijo / Head of Laboratory for Food Chemistry
Odsek za Analizno kemijo / Department of Analytical Chemistry

[KI]

[KI]



Kemijski inštitut / National Institute of Chemistry
Hajdrihova 19
SI-1000 Ljubljana
Slovenija / Slovenia
Telefon / Phone: 00386 1 476 02 69
***@***.******@***.***>     www.ki.si<http://www.ki.si/>



P Do you really have to print this e-mail? Think green.

**Comment by yesiman** [comment]
Good evening,
Apparently no problem your image is a bit big but a loader appears after
the page will reload by itself and the image should be visible.
Can you make another test ? just select your avatar image and wait for the
automatic reload of the page ?
Thanks.

Le lun. 11 oct. 2021 à 15:32, AlenAlbreht ***@***.***> a
écrit :

**Comment by AlenAlbreht** [comment]
I managed to upload the picture. One word of advice though... the picture does not resize automatically to fit the window on the webpage (portrait pictures are no good – half of my head was missing). So this is maybe something to fix in the future as not all members probably have portrait headshots. Also there seems to be the same issue with logos and actors.

Cheers
Alen

**Comment by slavenIBE** [comment]
Hello,
I have the same issue with the images even though the image is small and it is only 800 kb in size.
How should this be resolved?

**Comment by yesiman** [comment]
Hello,
800ko is big for an avatar image, but it should work, there was no issue on
this you only have to wait the end of the upload and the page will refresh
by itself (no need to click on validate button)
Thanks

**Comment by slavenIBE** [comment]
Unfortunately, the page refreshes itself but no image gets uploaded. It remains blank afterward. I even lowered the size of the photo under 800 kb

**Comment by yesiman** [comment]
Please send me your image.
Thanks
